crates/topos-api/tests/tce_layer.rs

```diff
@@ -255,3 +258,11 @@ async fn create_tce_layer() {
     drop(stream);
     jh.await.unwrap();
 }
+
+#[rstest]
+#[test(tokio::test)]
+async fn create_grpc_client() {
+    let entrypoint = Endpoint::from_static("http://127.0.0.1:1340").connect_lazy();
+
+    let _client = SynchronizerServiceClient::init(entrypoint);
+}
```

Comment:
Not sure this test adds much value?

Reply:
It tests that we can initialize a client with a channel using the custom-made trait. It also make the patch coverage happy 😃

Reply:
> It also make the patch coverage happy 😃

This is exactly why I am skeptical of coverage measurements. It can lead to adding tests that we wouldn't otherwise, just to satisfy the "bureaucracy".